src/index.js

```diff
@@ -60,6 +65,17 @@ const downloadImages = () => {
 };
 
 /**
+ * Make sure the filenames are safe by stripping out problematic characters
+ * @param {String} filename a potential filename
+ * @returns {String} filename with characters stripped out
+ */
+const cleanFileName = function (filename) {
```

Comment:
This is just in case the tumblr user/blog name has problematic characters in it. Since the blog name will be the same throughout, I could make it more efficient by only running it once on the blog name (rather than how I have it running for every post).